### ISSUE TITLE
Installing ERROR: No matching distribution found for gradio_client==0.2.5

### ISSUE THREAD

**Issue by vasrnet** [oobabooga/text-generation-webui#2239]
### Describe the bug

When run “start_windows.bat ”  for a moment，display an error：No matching distribution found for gradio_client==0.2.5
and then  run“update_windows.bat” ，this error Reappeared，Unable to complete installation.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1、run “start_windows.bat ” ,error displayed；
2、 run“update_windows.bat” error Reappeared

### Logs

```shell
Using cached https://pypi.tuna.tsinghua.edu.cn/packages/20/ae/a54960a728230f73432a547b07c9369a4c5d58d4e7b93402dbd7b8ffd9b8/flexgen-0.1.7-py3-none-any.whl (50 kB)
ERROR: Could not find a version that satisfies the requirement gradio_client==0.2.5 (from versions: 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6b10, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.1.0, 0.1.1, 0.1.2b1, 0.1.2, 0.1.3, 0.1.4, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4)
ERROR: No matching distribution found for gradio_client==0.2.5
Command '"C:\oobabooga_windows\installer_files\conda\condabin\conda.bat" activate "C:\oobabooga_windows\installer_files\env" >nul && python -m pip install -r requirements.txt --upgrade' failed with exit status code '1'. Exiting...

Done!
```

### System Info

```shell
NVIDIA GeForce RTX 3060
Intel(R) Xeon(R) CPU E5-2667 4 @ 3.20GHz
63.9 GB RAM
```

**Comment by xinxi053** [comment]
yes, i got the same error too.  ERROR: Could not find a version that satisfies the requirement gradio_client==0.2.5 (from versions: 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6b10, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.1.0, 0.1.1, 0.1.2b1, 0.1.2, 0.1.3, 0.1.4, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4)
ERROR: No matching distribution found for gradio_client==0.2.5


also  when i use conda install gradio command , it can't find the package:

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - gradio

Current channels:

  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by xinxi053** [comment]
https://pypi.org/project/gradio-client/0.2.5/#modal-close 
manually download the package, and pip install gradio-client0.2.5.tar.gz  skip this problem.

**Comment by vasrnet** [comment]
Yes manually download and another problem like this:https://github.com/oobabooga/text-generation-webui/issues/795

**Comment by vasrnet** [comment]
INFO:Loading anon8231489123_vicuna-13b-GPTQ-4bit-128g...
ERROR:Failed to load GPTQ-for-LLaMa
ERROR:See https://github.com/oobabooga/text-generation-webui/blob/main/docs/GPTQ-models-(4-bit-mode).md
